data-raw/render_mexico_codes.R

```diff
@@ -1,14 +1,79 @@
-# mexico region codes
-mexico_codes <- vroom::vroom("data-raw/mexico_codes.csv")
-mexico_codes <- tibble::tibble(
-  level = c("level_1_region", "level_2_region"),
-  name = c("iso_3166_2", "inegi_code"),
-  codes = list(
-    dplyr::select(mexico_codes, region_level_1, iso_code) %>%
-      unique(),
-    mexico_codes
-  )
-)
+# Set Mexico region codes
+# 
+# Level 1 codes: ISO-3166-2
+# Source: https://en.wikipedia.org/wiki/ISO_3166-2:MX
+# 
+# Level 2 codes: INEGI Mexican official statistics geocoding
+# Source: Raw data
+# Note: Level 1 INEGI codes are the first 2 characters of Level 2 INEGI codes
```

Comment:
Great stuff. Is this info mirrored in the Mexico class definition as most users probably won't find this script.